### ISSUE TITLE
Do not run when a footprint with no corresponding sibling footprints are found

### ISSUE THREAD

**Issue by EeliK** [MitjaNemec/ReplicateLayout#6]
Select a footprint in a sheet which doesn't have siblings. The plugin is run although it's not useful. A message should be shown which tells to select one footprint which has corresponding footprint(s) in other sheet(s).

I should try this again on Linux. I once had a situation where the plugin and whole KiCad were frozen when I selected a footprint in top level sheet. The plugin shows nothing in Source hierarchy level and Destinatioin sheets lists. I can't now replicate that problem on Windows, but the plugin can do nothing useful anyway with such footprint.

**Comment by MitjaNemec** [comment]
I agree. I'll get on it, but it will take a while, as the change will require a bit of refactoring to the backend code
